src/snowflake/snowpark/_internal/data_source/datasource_partitioner.py

```diff
@@ -69,6 +70,8 @@ def __init__(
         self.dialect = self.dialect_class()
         self.driver = self.driver_class(create_connection, dbms_type)
 
+        self._query_input_alias = f"SNOWPARK_DBAPI_QUERY_INPUT_ALIAS_{generate_random_alphanumeric(5).upper()}"
```

Comment:
the alias is only for query right? then should we do the following?

`self._query_input_alias = f".." if is_query else None`